### DISCUSSION TITLE
Custom font not loading in Vervel deployment

### DISCUSSION THREAD

**Discussion by rileyrichter** [vercel/community#2752] [Help] [answered]
### Summary

I have a project I deployed and you can view it here: 

- [Vercel URL](https://rock-paper-scissors-ten-beige.vercel.app/)
- [Custom domain](http://app.roshambo.dev)

In the local environment, my custom fonts load, but they don't on Vercel. I can see the fonts referenced in the CSS but in the network tab nothing is showing. 

### Steps to Reproduce

1. Load the site here: http://app.roshambo.dev
2. See the custom fonts aren't loading

**Comment by amyegan** [comment]
Please share a minimal reproduction, or your repo if it's public, so we can get a better understanding of what is causing the problem 🙏

**Reply by rileyrichter** [reply]
Thanks, @amyegan!! 

Here's more: 

- Repo: https://github.com/Webflow-Examples/rock-paper-scissors
- Live site: http://app.roshambo.dev/


Repro steps: 

1. navigate to http://app.roshambo.dev/
2. right click and [inspect the button in the navbar](https://share.cleanshot.com/PmZ41WR0)
3. note in the inspector that the [font should be Press Start 2P](https://share.cleanshot.com/4YpdKpGp)
4. [check the sources tab](https://share.cleanshot.com/G8nr8dRB), open css, check that the Google font has been imported 
5. visit https://fonts.google.com/specimen/Press+Start+2P to see that the font on the site is not Press Start 2P

I checked the Network tab in the dev tools and don't see the font loading, but when I run this locally [I do see the right fonts loading](https://share.cleanshot.com/5JPqyBQq). I'm trying to understand why they aren't loading when deployed to Vercel. 

Thanks in advance for any help and I appreciate you taking a look!

**Reply by amyegan** [reply, marked as answer]
Sorry for the delay. I wasn't able to replicate the problem in my other test apps, but I do see the font problem with the roshambo app. I was able to get the font to display in my test deployment of this code by adding the following import line to `Header.module.css`:
```css
@import url('https://fonts.googleapis.com/css2family=Press+Start+2P&family=Source+Sans+3:wght@400;700&display=swap');
```

**Comment by errorcodezero** [comment]
This also happens to me and I can't solve it for the life of me.

**Reply by amyegan** [reply]
Hi @errorcodezero. I'm not able to reproduce the issue in my own test apps, so it seems like a code problem rather than a platform issue. Please open a new discussion and share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) and someone here may be able to recognize the source of the problem. 🙏

**Reply by errorcodezero** [reply]
Alright I'm on it
